x-pack/plugins/observability/public/routes/index.tsx

```diff
@@ -113,6 +113,17 @@ export const routes = {
     },
     breadcrumb: [casesBreadcrumbs.cases],
   },
+  '/cases/:detailName/:commentId': {
```

Comment:
@smith We link to the comment as shown in the PR description GIF. Needed this for it to work in Observability or it is a broken link 🙅

Reply:
Shouldn't `commentId` also be in the `params.path` below also?

Reply:
It works without it, but if you tried to reference the comment id from the `useParams` in `CaseDetailsPage` it would fail the type check.

Reply:
got it, good catch